### ISSUE TITLE
accessibility features udpate

### ISSUE THREAD

**Issue by MathuraMG** [processing/p5.js-web-editor#672]
Some features need to be updated to support accessibility
* [Overlay buttons not labeled](https://github.com/processing/p5.js-web-editor/blob/master/client/modules/App/components/Overlay.jsx#L76). Reads as 'unlabeled button' on a screen reader
* Autofocus on the accessible-output is currently not working.

@catarak - would it be okay if @lm-n or myself assigned this to us?

#### Nature of issue?

- [ ] Found a bug
- [ ] Existing feature enhancement
- [ ] New feature request

#### Details about the bug: 

- Web browser and version: 
- Operating System: 
- Steps to reproduce this:

#### New feature details:

**Comment by catarak** [comment]
yes, go for it!

**Comment by kjhollen** [comment]
It looks like there are some ARIA labels in the code. Were they there previously, but not working? Or have they been added sometime in the last few years without updating the issue? (am doing some issue clean-up today and looking for things that can be closed) :)

**Comment by raclim** [comment]
Hi all! I currently tested this issue using Voiceover on a Macbook, and was overall able to hear the corresponding labels for the overlay buttons. I also felt the flow made sense in terms of the accessible outputs, but I could be missing some insight or perspective on this! 

Otherwise, I plan to close this issue in a few days. Please feel free to let me know if this should remain open, or create a new issue/reopen this one in the future, thanks so much!
